docs/manual-CN/2.query-language/4.statement-syntax/1.data-definition-statements/create-tag-edge-syntax.md

```diff
@@ -96,5 +96,5 @@ nebula> CREATE TAG icecream(made timestamp, temperature int)
 nebula> CREATE EDGE garbage (thrown timestamp, temperature int)
    TTL_DURATION = -2, TTL_COL = thrown,
    TTL_DURATION = 10, TTL_COL = thrown
```

Comment:
It's best not to write this syntax, it's easy to cause confusion

Reply:
line 92- line 98,  please use one ttl attribute

Reply:
ttl usage is as follows:
create tag (a int, b int. c string);
alter tag t add ttl_col = “a“， ttl_duration = 1000;  -- add ttl attribute
show create tag t;
alter tag t drop a；    -- drop ttl attribute   column
show create tag  t;
alter tag t add ttl_col = “b“， ttl_duration = 1000;
alter tag  t change (b string);    -- failed，column contail ttl attribute not to change
show create tag t;
alter tag t add ttl_col = ““；   -- drop ttl attribute
drop tag t;


create tag t(a int, b int. c string) ttl_duration= 100, ttl_col = "a";
Same as above

the edge is same as above.

Note that the tag or edge cannot have both the ttl attribute and the index, only one
create tag t(a int, b int. c string) ttl_duration= 100, ttl_col = "a";
create tag index id1 on t(a);     -- failed，

create tag t1(a int, b int. c string);
create tag index id1 on t1(a); 
alter tag t1 ttl_col = "a", ttl_duration = 100;       -- failed，